tests/e2e/specs/widgetPanel.js

```diff
@@ -154,4 +172,88 @@ describe("Widget panel usability test", () => {
     cy.get(`[data-cy=widget-delete-btn-yes]`).first().click({ force: true });
     cy.get(`[data-cy=widget-Misnamed]`).should("not.exist");
   });
+  it("Add Co2 widget & don't delete", () => {
```

Comment:
you create a widget that lets you add the percentage, but you don't test it. Test the widget in its entirety.